addons/main/functions/anomalies/fn_createTeleport.sqf

```diff
@@ -8,19 +8,37 @@
     Parameter:
         _pos - PositionASL where the anomaly should be (default: [0,0,0]])
         _id - ID which connects teleporters
+        _onEnterCode - code that is executed when a object enters this teleporter
+        _onExitCode - code that is executed when a object exits this teleporter
 
     Returns:
         Anomaly Trigger
 
     Author:
     diwako 2017-12-14
 */
-params[["_pos",[0,0,0]],["_id",-1]];
+params[["_pos", [0, 0, 0]], ["_id", -1], ["_onEnterCode", {}], ["_onExitCode", {}]];
 if !(isServer) exitWith {};
 
 if !(_pos isEqualType []) then {
     //created via module
-    _id = _pos getVariable ["anomalyid",-1];
+    _id = _pos getVariable [QGVAR(anomalyId), -1];
+    _onEnterCode = _pos getVariable [QGVAR(onEnterCode), ""];
+    _onExitCode = _pos getVariable [QGVAR(onExitCode), ""];
+
+    try {
+        _onEnterCode = compile _onEnterCode;
+    } catch {
+        _onEnterCode = {};
+        hintC ("On Enter Code failed to compile. Affected anomaly at " + str (getPosASL _pos));
+    };
+    try {
+        _onExitCode = compile _onExitCode;
+    } catch {
+        _onExitCode = {};
+        hintC ("On Exit Code failed to compile. Affected anomaly at " + str (getPosASL _pos));
+    };
```

Comment:
With the cba events, i doubt these are really needed anymore. A mission maker can easily just add an eventlistener at this point.

Reply:
Okay. I will revert it back.
As a personal preference, I like having module specific code. I like that there is a entry point in a module in 3den.